java/ql/src/experimental/Security/CWE/CWE-209/SensitiveDataExposureThroughErrorMessage.ql

```diff
@@ -0,0 +1,54 @@
+/**
+ * @name Information exposure through a error message
+ * @description Information from a error message propagates to an external user.
+ *              Error messages can unintentionally reveal implementation details
+ *              that are useful to an attacker for developing a subsequent exploit.
+ * @kind problem
+ * @problem.severity error
+ * @security-severity 5.4
```

Comment:
I am not sure if the security-severity should be the same. 

I tried to read up on how it is calculated but could not figure that out.